hbase-server/pom.xml

```diff
@@ -515,6 +515,33 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <artifactId>junit</artifactId>
+      <groupId>junit</groupId>
+    </dependency>
```

Comment:
Where is this junit dependency gets used?

Reply:
HBaseTestingUtility

Reply:
nit : Try to see if you can move that code out of src/java and move it in src/test if possible.